### PR TITLE
📖 Add Scaphandre project documentation

### PR DESCRIPTION
**Pull request by furkatgofurov7**
Adds [Scaphandre](https://github.com/hubblo-org/scaphandre) project user guide based on the energy efficiency study.

### CODE REVIEW

File: scaphandre/README.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+34.067745 W	1502751	"qemu-system-x86"
+0.275852 W	215	"ksmd"
+0.137926 W	1211	"containerd"
+0.137926 W	1274	"lldpd"
+0 W	1	"systemd"
+------------------------------------------------------------
+
+```
+
+Let's briefly describe what you see here. First Line is the power consumption of
```
Comment (fmuyassarov) [inline scaphandre/README.md:110]:
```suggestion
Let's briefly describe what you see here. First line is the power consumption of
```

**Review by fmuyassarov — DISMISSED** [review]

**Review by fmuyassarov — DISMISSED** [review]

File: scaphandre/README.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+	 Kubernetes cluster (to install Scaphandre, Prometheus and Grafana)
+
+### Option 1: Run scaphandre using docker
+
+This is the easiest and quickest way to try out the Scaphandre project using
+docker. For that, just run:
+
+```shell
+docker run -v /sys/class/powercap:/sys/class/powercap -v /proc:/proc -ti
+hubblo/scaphandre stdout -t 15
```
Comment (lentzi90) [inline scaphandre/README.md:86]: I think this should be without the line break, right?

```suggestion
docker run -v /sys/class/powercap:/sys/class/powercap -v /proc:/proc -ti \
  hubblo/scaphandre stdout -t 15
```

File: scaphandre/README.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+```
+
+You should get an output like:
+
+```bash
```
Comment (lentzi90) [inline scaphandre/README.md:91]: No need for `bash` here since it is just "random" text (not bash code).
````suggestion
```
````

**Review by lentzi90 — COMMENTED** [review]
Awesome!
Left a couple of nits.

**Review by lentzi90 — APPROVED** [review]

**Review by kashifest — APPROVED** [review]

**Pull request merged**